packages/example/README.md

```diff
@@ -1,34 +1,38 @@
 # Code example
 
-This directly contains a simple, framework-independent example for Connect-Web.
-It is a web interface for [ELIZA](https://en.wikipedia.org/wiki/ELIZA), a very
-simple natural language processor built in the 1960s to represent a 
-psychotherapist. The ELIZA service is provided by us at [demo.connect.build](https://demo.connect.build).
-You can find the protocol buffer schema [on the BSR](https://buf.build/bufbuild/eliza/tree/main:buf/connect/demo/eliza/v1/eliza.proto).
+This directory contains a simple example for running a Connect-Web client with a Connect-Node server using [Express](https://expressjs.com/).
+
+The application is a web interface for a stripped-down version of [ELIZA](https://en.wikipedia.org/wiki/ELIZA), a very
+simple natural language processor built in the 1960s to represent a psychotherapist. 
 
+You can find the protocol buffer schema [on the BSR](https://buf.build/bufbuild/eliza/tree/main:buf/connect/demo/eliza/v1/eliza.proto).
```

Comment:
Can we keep the local eliza.proto? Generating from a remote module is super convenient, but I think the ability to modify the proto file and see how the generated code changes overweighs here.

Reply:
Sure.  Added back.